extern/StormLib/src/StormPort.h

```diff
@@ -254,6 +254,34 @@
 
 #endif
 
+//-----------------------------------------------------------------------------
+// Defines for Wii U platform
+
+#if !defined(STORMLIB_PLATFORM_DEFINED) && defined(__WIIU__)
+
+  #include <sys/types.h>
+  #include <sys/stat.h>
+  #include <fcntl.h>
+  #include <unistd.h>
+  #include <stdint.h>
+  #include <stdlib.h>
+  #include <stdio.h>
+  #include <stdarg.h>
+  #include <string.h>
+  #include <strings.h>
+  #include <ctype.h>
+  #include <assert.h>
+  #include <errno.h>
+  #include <malloc.h>
+
+  #undef STORMLIB_LITTLE_ENDIAN                     // Wii U is always big endian
+
+  #define STORMLIB_MAC                              // Use Mac compatible code
+  #define STORMLIB_WIIU
+  #define STORMLIB_PLATFORM_DEFINED
+
+#endif
```

Comment:
@dcvz any chance we could try to get this PR'd upstream? or does it rely on stuff that's LUS/SoH specific?

Reply:
I'm pretty sure I saw a PR to StormLib, and it looked like this.

Reply:
ah yes, i see this was part of https://github.com/ladislav-zezula/StormLib/pull/267/files

perfect!